### DISCUSSION TITLE
AGENTS

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#7470] [Unsupported Versions] [answered]
Hey,

I am a bit confused about the agents.

Do we need to install winlogbeat,wazuh agent, and osquery agents together, or do we just install one of them?

Also, do we supposed to see Wazuh Manager at Kibana?

Regards.

**Comment by g4l1l30sec** [comment]
To send the logs to Kibana you can use Wazzuh or Winlogbeat/Filebeat. Osquery is for the fleet. Check out the Download at main page.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/host.html:
> You can send logs to Security Onion via your choice of either [osquery](https://docs.securityonion.net/en/2.3/osquery.html#osquery), [Beats](https://docs.securityonion.net/en/2.3/beats.html#beats), [Wazuh](https://docs.securityonion.net/en/2.3/wazuh.html#wazuh), or [Syslog](https://docs.securityonion.net/en/2.3/syslog.html#syslog).

**Reply by ekky19** [reply]
Thank you for the clear answer. 
So there is no need to install more than one host agent on one machine.

**Reply by yonikhil** [reply]
Do you ever install elastic agent in any host machine?
